pkg/cmd/render/render_test.go

```diff
@@ -109,3 +119,131 @@ func TestDiscoverCIDRs(t *testing.T) {
 		}
 	}
 }
+
+func TestRenderCommand(t *testing.T) {
+	assetsInputDir, err := ioutil.TempDir("", "testdata")
+	if err != nil {
+		t.Errorf("unable to create assets input directory, error: %v", err)
+	}
+	templateDir := filepath.Join("..", "..", "..", "bindata", "bootkube")
+
+	tests := []struct {
+		// note the name is used as a name for a temporary directory
+		name         string
+		args         []string
+		testFunction func(cfg *kubecontrolplanev1.KubeAPIServerConfig) error
+	}{
+		{
+			name: "scenario 1 checks feature gates",
+			args: []string{
+				"--asset-input-dir=" + assetsInputDir,
+				"--templates-input-dir=" + templateDir,
+				"--asset-output-dir=",
+				"--config-output-file=",
+			},
+			testFunction: func(cfg *kubecontrolplanev1.KubeAPIServerConfig) error {
+				actualGates, ok := cfg.APIServerArguments["feature-gates"]
+				if !ok {
+					return fmt.Errorf("missing \"feature-gates\" entry in APIServerArguments")
+				}
+				expectedGates := []string{
```

Comment:
get this from https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L68-L78 please

Reply:
sure, np.